client/src/components/Home.jsx

```diff
@@ -27,24 +27,30 @@ function Home (
       return (
         <div>
           {naviBar}
-          {toDoList}
+          <div className="">
```

Comment:
Why is a div with no class name being added? why not return this div from toDoList?